IdentityCore/src/oauth2/MSIDWebviewFactory.m

```diff
@@ -183,6 +186,14 @@ @implementation MSIDWebviewFactory
         [result addEntriesFromDictionary:allAuthorizeRequestExtraParameters];
     }
     
+#if MSAL_JS_AUTOMATION
+    // Remove "script" entry from the additional parameters
+    if([result objectForKey:@"script"])
```

Comment:
Can we call this parameter something more detailed as well? e.g. client_automation_script?

Reply:
@oldalton We are currently sending this "script" as the key to WAM for javascript injection. In order to keep it the same (it will be a general method for all the platforms), I prefer to still use it here.